app/pages/projects/search.tsx

```diff
@@ -447,26 +489,55 @@ export const Projects = () => {
                     </AccordionDetails>
                   </Accordion>
                 )}
-                {tierFacets.length > 0 && (
+                {missingFacets.length > 0 && (
                   <Accordion disableGutters className="homeWrapper__accordion">
                     <AccordionSummary
                       expandIcon={<ExpandMore />}
-                      aria-controls="panel3a-controls"
-                      id="panel3a-header"
+                      aria-controls="panel2a-controls"
+                      id="panel2a-header"
                       className="accordion__filter__title"
                     >
-                      <h4>Innovation tiers</h4>
+                      <h4>Missing Roles</h4>
                     </AccordionSummary>
                     <AccordionDetails>
                       <ul className="homeWrapper__myProposals--list">
-                        {tierFacets.map((item) => (
+                        {missingFacets.map((item) => (
                           <li key={item.name}>
                             <Link
                               id={item.name}
                               underline="none"
                               href=""
                               color="#AF2E33"
-                              onClick={(e) => goToSearchWithFilters(e, "tier")}
+                              onClick={(e) => goToSearchWithFilters(e, "missing")}
+                            >
+                              {item.name} ({item.count})
```

Comment:
Is it necessary to show the count for missing fields? i find it a bit confusing as the count is for the projects that have the role, so maybe it's better to display the count for projects without the role or just don't show any count

Reply:
I'm also on the fence for that one. At one point I was hiding it. I thought the info could be useful, and calculating the projects that don't have seemed hard to calculate. Or at least I lost the mental strength to try it 🙈 .

Now that you mention it. I agree with you, it confuses more than helps, and you can get the actual missing count by actually clicking and filtering :-p. I also agree that is better to hide it.

Done!